packages/manager/apps/hub/package.json

```diff
@@ -48,6 +47,7 @@
     "@ovh-ux/ng-ovh-sso-auth-modal-plugin": "^4.1.1",
     "@ovh-ux/ng-ovh-swimming-poll": "^5.0.5",
     "@ovh-ux/ng-ovh-user-pref": "^2.0.4",
+    "@ovh-ux/ng-shell-tracking": "^0.0.0",
```

Comment:
**question:** To meet to first release of `@ovh-ux/ng-shell-tracking` component, shouldn't be this?
```suggestion
    "@ovh-ux/ng-shell-tracking": "^0.0.0 || ^1.0.0",
```